ui/admin/app/templates/scopes/scope/users/index.hbs

```diff
@@ -14,94 +14,113 @@
       {{t 'resources.user.title_plural'}}
       <DocLink @doc='user' @iconSize='large' />
     </h2>
-    {{#if @model}}
+    {{#if @model.usersExist}}
       <p>{{t 'resources.user.description'}}</p>
     {{/if}}
   </page.header>
 
   <page.actions>
     {{#if (can 'create model' this.scope collection='users')}}
-      {{#if @model}}
-        <Rose::LinkButton @route='scopes.scope.users.new' @style='primary'>{{t
-            'resources.user.titles.new'
-          }}</Rose::LinkButton>
+      {{#if @model.usersExist}}
+        <Hds::Button
+          @route='scopes.scope.users.new'
+          @text={{t 'resources.user.titles.new'}}
+        />
```

Comment:
Why would we hide this if no users existed?

Reply:
great question, I also had the same question. It's because when we have none of a resource we display a welcome message instead that has a new link for creation. We also do this in other resources (we hid new button and description) 
![image](https://github.com/hashicorp/boundary-ui/assets/107949262/895634b1-33ea-4c9d-b456-229dc5221b11)

Reply:
OOOOOH that makes total sense. Thanks for the explanation